.script/utils/schemas/workbooksMetadataSchema.json

```diff
@@ -58,7 +58,82 @@
                 },
                 "featureFlag": {
                     "type": "string"
+                },
+                "source": {
+                    "type": "object",
+                    "properties": {
+                        "kind": {
+                            "type": "string"
+                        },
+                        "name": {
+                            "type": "string"
+                        },
+                        "sourceId": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "author": {
+                    "type": "object",
+                    "properties": {
+                        "name": {
+                            "type": "string"
+                        },
+                        "email": {
+                            "type": "string"
+                        },
+                        "link": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "support": {
+                    "type": "object",
+                    "properties": {
+                        "tier": {
+                            "type": "string"
+                        },
+                        "name": {
+                            "type": "string"
+                        },
+                        "email": {
+                            "type": "string"
+                        },
+                        "link": {
+                            "type": "string"
+                        }
+                    }
+                },
+                "categories": {
+                    "type": "object",
+                    "properties": {
+                        "domains": {
+                            "type": "array",
+                            "items": [
+                                { "type": "string" }
+                            ]
+                        },
+                        "verticals": {
+                            "type": "array",
+                            "items": [
+                                { "type": "string" }
+                            ]
+                        }
+                    }
+                },
+                "threatAnalysisTactics": {
+                    "type": "array",
```

Comment:
There's also dark images

Reply:
in workbooks there is a naming convention in the `previewImageFileNames` if they have White in them - its white, if Black - its black. the scripts separates it into dark and light based on this naming